src/renderers/webgl/WebGLCapabilities.js

```diff
@@ -22,6 +24,33 @@ function WebGLCapabilities( gl, extensions, parameters ) {
 
 	}
 
+	function canReadPixelFormat( textureFormat ) {
```

Comment:
Nomenclature suggestion:
```js
textureFormatReadable( textureFormat ) // or pixelFormatReadable( textureFormat )

textureTypeReadable( textureType )
```

Reply:
Renamed the methods to `textureTypeReadable` and  `textureFormatReadable`, and squashed the commits.